src/css/controls/imports/icons.less

```diff
@@ -68,6 +68,24 @@
     }
 }
 
+.jw-icon-pip {
+    .jw-svg-icon-pip-off {
+        display: none;
+
+        .jw-off& {
+            display: block;
```

Comment:
I may be misunderstanding, but it doesn't look like the selector being styled is what you're trying to style.

<img width="269" alt="image" src="https://user-images.githubusercontent.com/2903955/122102791-6f134d80-cddb-11eb-9efb-1415fa3aa99f.png">

Reply:
Also, jw-off is typically reserved for hiding icons via `display: none`, may want to change the className to `jw-on` or `jw-visible`

Reply:
Please see below

Reply:
Ah, makes sense, looks like a vs code issue on my side.